examples/nirfmxwlan/ofdm-modacc.py

```diff
@@ -0,0 +1,438 @@
+                selector_string="",
+                measurement_offset=0,
+                maximum_measurement_length=16,
+            )
+        )
+    )
+    raise_if_error(
+        client.OFDMModAccCfgFrequencyErrorEstimationMethod(
+            nirfmxwlan_types.OFDMModAccCfgFrequencyErrorEstimationMethodRequest(
+                instrument=instr,
+                selector_string="",
+                frequency_error_estimation_method=nirfmxwlan_types.OFDM_MODACC_FREQUENCY_ERROR_ESTIMATION_METHOD_PREAMBLE_AND_PILOTS,
+            )
+        )
+    )
+    raise_if_error(
+        client.OFDMModAccCfgAmplitudeTrackingEnabled(
+            nirfmxwlan_types.OFDMModAccCfgAmplitudeTrackingEnabledRequest(
+                instrument=instr,
+                selector_string="",
+                amplitude_tracking_enabled=nirfmxwlan_types.OFDM_MODACC_AMPLITUDE_TRACKING_ENABLED_FALSE,
+            )
+        )
+    )
+    raise_if_error(
+        client.OFDMModAccCfgPhaseTrackingEnabled(
+            nirfmxwlan_types.OFDMModAccCfgPhaseTrackingEnabledRequest(
+                instrument=instr,
+                selector_string="",
+                phase_tracking_enabled=nirfmxwlan_types.OFDM_MODACC_PHASE_TRACKING_ENABLED_TRUE,
+            )
+        )
+    )
+    raise_if_error(
+        client.OFDMModAccCfgSymbolClockErrorCorrectionEnabled(
+            nirfmxwlan_types.OFDMModAccCfgSymbolClockErrorCorrectionEnabledRequest(
+                instrument=instr,
+                selector_string="",
+                symbol_clock_error_correction_enabled=nirfmxwlan_types.OFDM_MODACC_SYMBOL_CLOCK_ERROR_CORRECTION_ENABLED_TRUE,
+            )
+        )
+    )
+    raise_if_error(
+        client.OFDMModAccCfgChannelEstimationType(
+            nirfmxwlan_types.OFDMModAccCfgChannelEstimationTypeRequest(
+                instrument=instr,
+                selector_string="",
+                channel_estimation_type=nirfmxwlan_types.OFDM_MODACC_CHANNEL_ESTIMATION_TYPE_REFERENCE,
+            )
+        )
+    )
+    raise_if_error(
+        client.OFDMModAccCfgAveraging(
+            nirfmxwlan_types.OFDMModAccCfgAveragingRequest(
+                instrument=instr,
+                selector_string="",
+                averaging_enabled=nirfmxwlan_types.OFDM_MODACC_AVERAGING_ENABLED_FALSE,
+                averaging_count=10,
+            )
+        )
+    )
+    raise_if_error(
+        client.Initiate(
+            nirfmxwlan_types.InitiateRequest(instrument=instr, selector_string="", result_name="")
+        )
+    )
+
+    ### Fetch Results ###
+
+    ofdm_mod_acc_fetch_composite_rmsevm_response = raise_if_error(
+        client.OFDMModAccFetchCompositeRMSEVM(
+            nirfmxwlan_types.OFDMModAccFetchCompositeRMSEVMRequest(
+                instrument=instr, selector_string="", timeout=10.0
+            )
+        )
+    )
+    composite_rmsevm_mean = ofdm_mod_acc_fetch_composite_rmsevm_response.composite_rms_evm_mean
+    composite_data_rmsevm_mean = (
```

Comment:
Style: I dont think the parenthesis add much here? I am assuming this is because of line length?

Reply:
Yeah, `black` added the parentheses so it could break apart the line.